spire/templates/apps/dovetail-marketing-form-handler.yml

```diff
@@ -107,6 +107,9 @@ Resources:
                 const email = params.get('inputEmail');
                 const name = params.get('inputName') || 'Unknown';
                 const organization = params.get('textOrganization') || 'Unknown';
+                const type = params.get('messageType') || 'Unknown';
```

Comment:
I'm using this for both the contact and optout requests, so I introduced a type to specify that, mostly for including in the ZD ticket subject to know how to handle the request more easily